lib/getstats.js

```diff
@@ -316,6 +320,41 @@ function chromeGetTrackStats(peerConnection, track) {
   });
 }
 
+/**
+ * Get the standardized statistics for a particular MediaStreamTrack in Edge.
+ * @param {RTCPeerConnection} peerConnection
+ * @param {MediaStreamTrack} track
+ * @returns {Promise.<StandardizedTrackStatsReport>}
+ */
+function edgeGetTrackStats(peerConnection, track) {
+  return new Promise(function(resolve, reject) {
```

Comment:
@syerrapragada This can be:
```js
return waitForEdgeIceCompletion(peerConnection).then(function() {
  return peerConnection.getStats(track);
}).then(function(response) {
  return standardizeEdgeAndFirefoxStats(response);
});
```

NOTE: Rename `standardizeFirefoxStats` to `standardizeEdgeAndFirefoxStats`.